src/main/java/life/qbic/io/commandline/AuthenticationOptions.java

```diff
@@ -76,4 +114,42 @@ public String toString() {
         .toString();
     //ATTENTION: do not expose the password here!
   }
+
+  public static class ReadProperties {
+
+    public static TreeMap<String, String> getProperties(String infile) {
+      final int lhs = 0;
+      final int rhs = 1;
+
+      TreeMap<String, String> map = new TreeMap<String, String>();
```

Comment:
NIT: more descriptive 
```suggestion
      TreeMap<String, String> properties = new TreeMap<String, String>();
```